worker/src/db_handler.rs

```diff
@@ -42,4 +80,28 @@ impl DBHandler {
             }
         });
     }
+
+    pub fn send_data_to_enclave(eid: sgx_enclave_id_t) -> Result<(), PolkadexDBError> {
+        let balances = load_balances_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+        let nonce = load_nonce_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+        let orderbook = load_orderbook_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+        enclave_send_disk_data(
```

Comment:
I think you might have to split the data into reasonable sized bits, like it's done [here](https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/develop/worker/src/main.rs#L640), otherwise the enlcve will most certainly run out of memory once some data has been stored in disk.

Reply:
Chunks loop implemented.